### PR TITLE
add taskId metadata

### PR DESCRIPTION
**Pull request by jmhoeft**
the label looks like this: ` "com.amazonaws.ecs.task-arn": "arn:aws:ecs:us-east-1:056684691971:task/dev-webapp/e43d1e93ae14461e819625d440eb8572",`

This should find that and add `{"taskId": "e43d1e93ae14461e819625d440eb8572"}` to metadata, right?!

### CODE REVIEW

File: bridge/util.go [hunk 1]
```diff
@@ -80,6 +80,11 @@ func serviceMetaData(config *dockerapi.Config, port string) (map[string]string,
 				metadata[key] = kvp[1]
 			}
 		}
+		if (kvp[0] == "com.amazonaws.ecs.task-arn") && len(kvp) > 1 {
+			key := "taskId"
+			value := string.Split(kvp[1], "/")
+	        metadata[key] = value[len(value)-1]
```
Comment (jesselang) [inline bridge/util.go]: Looks like you want to use `strings.SplitN()`. Should make `value` exactly what you want.

**Review by jesselang — CHANGES_REQUESTED** [review]
Looks like you want the `strings` package, not `string`.

**Review by tolson-vkn — APPROVED** [review]
Take it easy Rob Pike.

**Review by tolson-vkn — CHANGES_REQUESTED** [review]
Ope yeah it not string, it strings.

File: bridge/util.go [hunk 2]
```diff
@@ -80,6 +80,11 @@ func serviceMetaData(config *dockerapi.Config, port string) (map[string]string,
 				metadata[key] = kvp[1]
 			}
 		}
+		if (kvp[0] == "com.amazonaws.ecs.task-arn") && len(kvp) > 1 {
+			key := "taskId"
+			value := strings.SplitN(value, "/", -1)[2]
```
Comment (jesselang) [inline bridge/util.go]: You probably want to split `kvp[1]` yeah?

```suggestion
			value := strings.SplitN(kvp[1], "/", 3)[2]
```

**Review by jesselang — CHANGES_REQUESTED** [review]

File: bridge/util.go (hunk 2 above)
Comment (jmhoeft) [reply bridge/util.go]: Good call :facepalm:

**Review by jesselang — APPROVED** [review]
🚀

**Pull request merged**